examples/build.gradle

```diff
@@ -28,6 +28,11 @@ subprojects {
         }
     }
 
+    dependencies {
```

Comment:
I would add these in the projects, not top level, to reduce the drift for a user wanting to copy-paste this example out for their use.